aptos-move/framework/aptos-framework/sources/permissioned_signer.spec.move

```diff
@@ -113,31 +113,35 @@ spec aptos_framework::permissioned_signer {
 
     spec check_permission_exists<PermKey: copy + drop + store>(s: &signer, perm: PermKey): bool {
         pragma verify = false;
-        // pragma opaque;
-        // aborts_if false;
-        // ensures [abstract] result == spec_check_permission_exists(s, perm);
+        pragma opaque;
+        modifies global<PermissionStorage>(spec_permission_address(s));
+        ensures [abstract] result == spec_check_permission_exists(s, perm);
     }
 
-    spec fun spec_check_permission_exists<PermKey: copy + drop + store>(s: signer, perm: PermKey): bool {
-        use aptos_std::type_info;
-        use std::bcs;
-        let addr = spec_permission_address(s);
-        let key = Any {
-            type_name: type_info::type_name<PermKey>(),
-            data: bcs::serialize(perm)
-        };
-        if (!spec_is_permissioned_signer(s)) { true }
-        else if (!exists<PermissionStorage>(addr)) { false }
-        else {
-            // ordered_map::spec_contains_key(global<PermissionStorage>(addr).perms, key)
-            // FIXME: ordered map spec doesn't exist yet.
-            true
-        }
-    }
+    spec fun spec_check_permission_exists<PermKey: copy + drop + store>(s: signer, perm: PermKey): bool;
+
+
+    // spec fun spec_check_permission_exists<PermKey: copy + drop + store>(s: signer, perm: PermKey): bool {
```

Comment:
Is there a reason to keep this commented out code here? If so, could we add a bit of a comment on top (to be proven later after blah blah is fixed, or something similar)?